machine/machine_test.go

```diff
@@ -103,6 +115,10 @@ func TestNewAndLoadLinuxWithVmlinux(t *testing.T) { // nolint:paralleltest
 	testNewAndLoadLinux(t, "../vmlinux", "tap1", "192.168.30.1", "192.168.30.2", "24")
 }
 
+func TestNewAndLoadLinuxPVH(t *testing.T) { // nolint:paralleltest
+	testNewAndLoadLinux(t, "../vmlinux_PVH", "tap1", "192.168.30.1", "192.168.30.2", "24")
```

Comment:
If we make the tap device name and ip address unique, the test should succeed. 

```suggestion
	testNewAndLoadLinux(t, "../vmlinux_PVH", "tap2", "192.168.40.1", "192.168.40.2", "24")
```

Reply:
done